src/factories/EmployeeFactory.java

```diff
@@ -1,33 +0,0 @@
-package factories;
```

Comment:
👍 haha here is the removal of the factory!  I was just harping about that in the last PR.  Good to see you tackled it here with out me even saying anything to you.